### PR TITLE
Bug 1233355 - Add settings permissions to Sync app so it can use wind…

### PR DESCRIPTION
**Pull request by ferjm**
…ow.DUMP. r=michielbdejong

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0a6a42659854e8cafdec71389b30b321091ed852

**Comment by michielbdejong** [comment]
Isn't there a more elegant way to do this? IMHO logging is quite basic and should be unrelated to reading/writing settings. If only apps that read/write settings are allowed to write to the log then we should at least document that fact on https://developer.mozilla.org/en-US/Apps/Build/App_permissions#settings?

File: apps/sync/manifest.webapp [hunk 1]
```diff
@@ -8,6 +8,9 @@
      "name": "The Gaia Team",
      "url": "https://github.com/mozilla-b2g/gaia"
   },
+  "permissions": {
+    "settings": { "access": "readwrite" }
```
Comment (michielbdejong) [inline apps/sync/manifest.webapp]: please add `"description": "Necessary to see if logging is enabled"` and change to `readonly` as discussed on irc

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b01299269907c0516689f621a6d8e197a4e79ad7

**Pull request merged**